src/sentry/static/sentry/app/views/settings/components/settingsNavigation.tsx

```diff
@@ -6,6 +6,8 @@ import space from 'app/styles/space';
 import SettingsNavigationGroup from 'app/views/settings/components/settingsNavigationGroup';
 import {NavigationProps, NavigationSection} from 'app/views/settings/types';
 
+const FOOTER_HEIGHT = 92;
```

Comment:
shouldn't it be 93?

Reply:
Because of the border?

Do you see a scroll bar at 92?

Reply:
because the footer height is 93 😉

Reply:
Turns out the top spacing was 1px too large :)